src/app/public/modules/datepicker/datepicker-config.service.ts

```diff
@@ -2,8 +2,6 @@ import {
   Injectable
 } from '@angular/core';
 
-import 'moment/min/locales.min';
```

Comment:
☝️ We made this change for the RC-4 branch, but forgot to do it to the master branch.